packages/api/src/router/medias/media-router.ts

```diff
@@ -0,0 +1,88 @@
+import { TRPCError } from "@trpc/server";
+
+import { and, createId, desc, eq, like } from "@homarr/db";
+import { medias } from "@homarr/db/schema/sqlite";
+import { validation, z } from "@homarr/validation";
+
+import { createTRPCRouter, protectedProcedure } from "../../trpc";
+
+export const mediaRouter = createTRPCRouter({
+  getPaginated: protectedProcedure
+    .input(
+      validation.common.paginated.and(
+        z.object({ showAll: z.boolean().default(false), search: z.string().trim().default("") }),
+      ),
+    )
+    .query(async ({ ctx, input }) => {
+      const includeAll = ctx.session.user.permissions.includes("admin") && input.showAll;
```

Comment:
I dislike the name. Suggestion: ``includeFromAllUsers`` or ``includeOnyFromCurrentUser``